osc-ui/src/main/java/org/osc/core/broker/view/vc/securitygroup/BindSecurityGroupWindow.java

```diff
@@ -418,9 +413,10 @@ private void serviceTableClicked(long itemId) {
 		}
 	}
 
-	private boolean isMultiplePoliciesSupported(Long vsId) {
+	private boolean isBindedWithMultiplePolicies(Long vsId) {
 		for (VirtualSystemPolicyBindingDto binding : this.allBindings) {
```

Comment:
@arvindn05 we iterate on this.allBindings to find the right binding. If we want to introduce change, we will have to update serviceTableClicked to find the right bind and provide to isBindedWithMultiplePolicies

Reply:
i dont see the issue...you will still iterate in the isBindedWithMultiplePolicies(Long vsId) method...so no changes should be needed.